src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -77,6 +77,9 @@
             <span class="invalid-feedback"
                   *ngIf="serviceForm.showError('service_id', frm, 'required')"
                   i18n>This field is required.</span>
+            <span class="invalid-feedback"
+                  *ngIf="serviceForm.showError('service_id', frm, 'uniqueName')"
+                  i18n>The chosen service id is already in use.</span>
```

Comment:
For consistency with the other messages, "this x" or "the value":
```suggestion
                  i18n>This service id is already in use.</span>
```

Reply:
Done